manim/mobject/three_dimensions.py

```diff
@@ -136,3 +137,70 @@ def generate_points(self):
         Cube.generate_points(self)
         for dim, value in enumerate(self.dimensions):
             self.rescale_to_fit(value, dim, stretch=True)
+
+
+class Honeycomb(Tiling):
+    """
+    Inherits from Tiling and works effectively the same, just adding
+    the third dimension.
+    To achieve this __init__ and apply_transforms are extended,
+    while tile_init_loop is overridden.
+    Since it's a 3D honeycomb also allows for tile_dictionary to be
+    alternatively called as cell_dictionary.
+    See Tiling for more details.
+
+    Parameters
+    ----------
+    tile_prototype : Mobject or function(x,y,z) that returns a Mobject
+    x_offset : nested list of Mobject methods and values
+    y_offset : nested list of Mobject methods and values
+    z_offset : nested list of Mobject methods and values
+    x_range : range
+    y_range : range
+    z_range : range
+    
+    Example
+    -------
+    Honeycomb(Cube(),
+              [[Mobject.shift,[2.1,0,0]]],
+              [[Mobject.shift,[0,2.1,0]]],
+              [[Mobject.shift,[0,0,2.1]]],
+              range(-1,1),
+              range(-1,1),
+              range(-1,1))
```

Comment:
(Note that `::` alone does not print a `:` in docs - it just indicates highlighting. When, however, preceded by text, it prints a `:`.)
```suggestion
    ::
    
        Honeycomb(Cube(),
                  [[Mobject.shift,[2.1,0,0]]],
                  [[Mobject.shift,[0,2.1,0]]],
                  [[Mobject.shift,[0,0,2.1]]],
                  range(-1,1),
                  range(-1,1),
                  range(-1,1))
```